Elements/src/Model.cs

```diff
@@ -173,7 +173,7 @@ public string ToJson(bool indent = false)
             }
             exportModel.Transform = this.Transform;
 
-            return Newtonsoft.Json.JsonConvert.SerializeObject(this,
+            return Newtonsoft.Json.JsonConvert.SerializeObject(exportModel,
```

Comment:
Good catch. This was the result of some work in the representations branch that got reversed, but not fully.